src/Conf.ml

```diff
@@ -842,6 +842,46 @@ let output =
       & opt (some string) default
       & info ["o"; "output"] ~doc ~docs ~docv)
 
+let ocp_indent_options =
+  [ ("base", None)
+  ; ("type", None)
+  ; ("in", None)
+  ; ("with", None)
+  ; ( "match_clause"
+    , Some
+        ( "cases-exp-indent"
+        , "$(b,match-clause) is equivalent to $(b,cases-exp-indent)."
```

Comment:
Later, I expect that we will get into the situation where an ocp-indent option `foo` is used to set the ocamlformat option `bar`, but they are not exactly "equivalent". So I wonder whether we should phrase this as `$(b,match-clause) sets $(b,cases-exp-indent) to the same value.` instead?